.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+    "printWidth”: 120,“
```

Comment:
Out of curiosity is it working correctly with those Unicode quote marks `“` and `“` ?

Reply:
No, actually I overlooked it, and  changed it into regular " " marks.